backend/src/main/java/com/bakdata/conquery/models/config/ConqueryConfig.java

```diff
@@ -112,14 +113,14 @@
 
 
 	//this is needed to force start the REST backend on /api/
-	public ConqueryConfig() {
-		((DefaultServerFactory) this.getServerFactory()).setJerseyRootPath("/api/");
+	{
+		((DefaultServerFactory) getServerFactory()).setJerseyRootPath("/api/");
 	}
 
 	@Override
 	public void setServerFactory(ServerFactory factory) {
 		super.setServerFactory(factory);
-		((DefaultServerFactory) this.getServerFactory()).setJerseyRootPath("/api/");
+		((DefaultServerFactory) getServerFactory()).setJerseyRootPath("/api/");
 	}
```

Comment:
Das Festlegen auf `/api/` würde ich aus der Config raus lassen und in die `ApiV1.java` verlagern, da es hier sonst nichts zu konfigurieren gibt. Also so wie es im AdminServlet gemacht wird.